### DISCUSSION TITLE
Исходный код виджета

### DISCUSSION THREAD

**Discussion by Oihso** [cdek-it/widget#10] [Ideas]
Планируется ли добавить в репозиторий исходный код виджета, а не только скомпилированный JS код? Это позволило бы оптимизировать код виджета и ускорить разработку/скорость исправления ошибок

**Comment by vermorag** [comment]
Передал вопрос на согласование руководству, ответ сообщу до конца следующей недели.

**Comment by vermorag** [comment]
К сожалению, потребуется чуть больше времени - нужно получить согласование от нашей службы безопасности. Поскольку виджет полностью переписан с нуля и это новый проект - это обязательно. Сообщу новую дату как от них получу информацию.
Если есть какие-то конкретные вопросы или предложения по коду - могу на них ответить.

**Comment by nexoos-team** [comment]
Поддерживаю, добавьте исходный код виджета, приходится делать правки в минимизированной версии.

**Comment by piton-zel** [comment]
Шло время..., а религия так называемой службы безопасности так и не позволяет выложить исходники? Что такого секретного в данном коде, что его ни в коем случае нельзя засветить в исходниках, а разрешено только в минимизированной версии? По сути это тот же исходник, только с кривыми названиями, или это такой хитрый способ чтобы максимально усложнить общение между вендором и разработчиками? ;)

**Comment by Oihso** [comment]
@vermorag есть ли обновления по данному вопросу?

**Reply by vermorag** [reply]
Добрый день. Код 3 версии опубликован не будет.
Сейчас ведется работа над 4 версией и ее код будет раскрыт вместе с первым релизом.

**Reply by Oihso** [reply]
Отличные новости!
